City-of-Helsinki/drupal-helfi-rekry
# UHF-5287 Job listing page styling

# Job listing page styling [UHF-5287](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5287)
Styled the job listing page with information that was available on the content type and layouts.

## What was done
* hdbt_subtheme was enabled and updated on the rekry-instance
* Job listing page dom was constructed and styled.
* Organization information block for sidebar was created to helfi_rekry_content module and enabled in the configuration.

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT theme
    * `composer require drupal/hdbt:dev-UHF-5287_calendar_icon`
* Checkout this branch.
    * `git checkout UHF-5287_job_listing_design`
* Run `make drush-cr && make drush-cim && make drush-cr`

## How to test
* If you don't have any rekry-instance content available create one job listing content and view it. Basically fill all the information first and save the node and check that the page follows the designs. 
* You can later remove some of the content and make sure that the page reacts nicely to the missing data as well.
* Mobile: https://share.goabstract.com/ab3436e3-1b54-48be-a583-fa6253f72821?collectionLayerId=533a81d5-4e25-435a-85e4-977de64be81f&mode=build&sha=latest
* Desktop: https://share.goabstract.com/ab3436e3-1b54-48be-a583-fa6253f72821?collectionLayerId=da122a5c-c151-40a7-9dab-2fb3ddc622f0&mode=design&sha=latest

## Designers review

One of the checkboxes below needs to be checked (`[x]`)

* [x] This PR has been visually reviewed by a designer (Sami Haikonen)

## Other PRs
* https://github.com/City-of-Helsinki/drupal-hdbt/pull/339
